Gemfile

```diff
@@ -69,7 +69,11 @@ group :test do
 end
 
 # Tooling
-gem "airbrake", "~> 4.0"
+# NB: Skylight needs to come before airbrake, as they both try to monkey patch
+# net_http to observe performance, but doing it in the wrong order causes an
+# infinite loop
+gem "skylight"
+gem "airbrake"
```

Comment:
Huh, I think I ran into that too with something else! I'm not sure how I fixed it 🤔